dashboard/src/app/projects/create-project/create-project.controller.ts

```diff
@@ -1067,14 +973,14 @@ export class CreateProjectController {
       // generate a name
 
       // starts with project
-      var name = 'project';
+      let name = 'project';
```

Comment:
ok, up to you. In that case it's obvious but sometimes we declare variable used later.